app/src/components/model/modals/ValidateBeforeReview.js

```diff
@@ -0,0 +1,186 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  Button,
+  Box,
+  Typography,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+  CircularProgress,
+  DialogActions,
+} from "@mui/material";
+
+import CloseIcon from "@mui/icons-material/Close";
+import FactCheckIcon from "@mui/icons-material/FactCheck";
+import { useDispatch } from "react-redux";
+import { useModelID } from "../hooks/useModelID";
+import { useValidateQuery } from "../../../api/gram/validation";
+import { modalActions } from "../../../redux/modalSlice";
+import { MODALS } from "../../elements/modal/ModalManager";
+
+export function ValidateBeforeReview() {
```

Comment:
Consider renaming this to QualityCheck or similar instead, makes it easier to find.